modules/caddyhttp/caddyauth/basicauth.go

```diff
@@ -39,6 +39,8 @@ type HTTPBasicAuth struct {
 	HashRaw json.RawMessage `json:"hash,omitempty" caddy:"namespace=http.authentication.hashes inline_key=algorithm"`
 
 	// The list of accounts to authenticate.
+	// After a successful authentication, the placeholder
+	// `{http.auth.user.id}` will be set to the username.
```

Comment:
This should actually go in caddyauth.go as a comment on the Authentication type, since it doesn't matter what auth backend validates the user.